src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-bar__big"
+      method="get"
+      action="https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md"
+      data-qa="big"
+    >
+      <button class="btn__big" type="submit">
+        <img src="images/Search.svg"
+           alt="bigger-search">
+      </button>
```

Comment:
Use background-image instead

Reply:
I've already tried everything, but I can't make background . I looked through the background and did only focus